frontend/ui/widget/textwidget.lua

```diff
@@ -201,6 +201,49 @@ function TextWidget:_measureWithXText()
     end
 end
 
+function TextWidget:getFittedText()
+    -- Returns the substring of text that fits in self.max_width, without
```

Comment:
Looks like LDoc-type documentation to me, for in http://koreader.rocks/doc/modules/ui.widget.textwidget.html

Like one of these molds:
https://github.com/koreader/koreader/blob/b111ccc2b803fc78425d9791fcc8bb7a9a704e6e/frontend/util.lua#L27-L47
https://github.com/koreader/koreader/blob/b111ccc2b803fc78425d9791fcc8bb7a9a704e6e/frontend/util.lua#L948-L953

Reply:
Yeah, but none of the other methods have a ldoc doc. Feels awkward to document such a minor one and not have the other essential ones not documented :) But I don't feel like spending the time needed to document the other ones...

Reply:
Alright, just keep the comment as is then, but I think it'd be clearer above the function. (Also you could change the first line to three dashes to convert it into an LDoc comment in that case, resulting in a smaller diff.)